conf/app.routes

```diff
@@ -10,7 +10,8 @@ GET         /not-authorised                        uk.gov.hmrc.nisp.controllers.
 GET         /account                               uk.gov.hmrc.nisp.controllers.StatePensionController.show
 GET         /account/pta                           uk.gov.hmrc.nisp.controllers.StatePensionController.pta
 GET         /account/cope                          uk.gov.hmrc.nisp.controllers.StatePensionController.showCope
-GET         /signout                               uk.gov.hmrc.nisp.controllers.StatePensionController.signOut
+GET         /signout                               uk.gov.hmrc.nisp.controllers.StatePensionController.questionnaire
+GET         /sign-out                              uk.gov.hmrc.nisp.controllers.StatePensionController.signOut
```

Comment:
could we think of something other than two ways of writing sign out? Have a chat with @fred-jackson because he has signoff or sign-off on that sort of thing.